### PR TITLE
Fix build

### PR DESCRIPTION
**Pull request by tetov**
Updated package.xml and Dockerfile to fix some issues stemming from kinetic EOL while also working with noetic. Base container updated to noetic in 2nd commit.

### CODE REVIEW

**Comment by tetov** [comment]
Fixes half of #8.

File: Dockerfile [hunk 1]
```diff
@@ -34,12 +34,13 @@ ADD . $CATKIN_WS/src/compas_rrc_driver
...
-    && rm -rf /etc/ros/rosdep/sources.list.d/* \
-    && rosdep init && rosdep update \
     # Update apt-get because its cache is always cleared after installs to keep image size down
     && apt-get update \
-    && sudo apt-get install build-essential -y \
+    && test $ROS_PYTHON_VERSION -eq 3 && ROSDEP_PKG="python3-rosdep" || ROSDEP_PKG="python-rosdep" \
+    ; apt-get install build-essential $ROSDEP_PKG -y  \
+    # Reconfigure rosdep
+    && rm -rf /etc/ros/rosdep/sources.list.d/* \
+    && rosdep init && rosdep update --include-eol-distros \
```
Comment (gonzalocasas) [inline Dockerfile:43]: > include-eol-distros

oh, what a cruel reminder :P

**Review by gonzalocasas — APPROVED** [review]
LGTM!

**Comment by gonzalocasas** [comment]
Thanks!

**Pull request merged**